Source/WebCore/platform/network/ResourceResponseBase.h

```diff
@@ -297,6 +306,30 @@ struct ResourceResponseData {
     ResourceResponseData() = default;
     ResourceResponseData(ResourceResponseData&&) = default;
     ResourceResponseData& operator=(ResourceResponseData&&) = default;
+#if ENABLE(LOCAL_NETWORK_ACCESS)
+    ResourceResponseData(URL&& url, String&& mimeType, long long expectedContentLength, String&& textEncodingName, int httpStatusCode, String&& httpStatusText, String&& httpVersion, HTTPHeaderMap&& httpHeaderFields, std::optional<NetworkLoadMetrics>&& networkLoadMetrics, ResourceResponseSource source, ResourceResponseBaseType type, ResourceResponseBaseTainting tainting, bool isRedirected, UsedLegacyTLS usedLegacyTLS, WasPrivateRelayed wasPrivateRelayed, String&& proxyName, bool isRangeRequested, std::optional<CertificateInfo> certificateInfo, std::optional<IPAddressSpace> ipAddressSpace)
```

Comment:
This is a lot of duplicate code.  It might be nicer to just add one parameter protected by the ENABLE macro.  Same with RequestData.